docs/resources/rds_read_replica_instance.md

```diff
@@ -0,0 +1,119 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud\_rds\_read\_replica\_instance
+
+Manage RDS Read Replica Instance resource.
+
+## Example Usage
+
+### Create a Rds read replica instance
+```hcl
+resource "huaweicloud_networking_secgroup" "secgroup" {
+  name          = "terraform_test_sg_for_rds"
+  description   = "security group for rds read replica instance test"
+}
+
+resource "huaweicloud_rds_instance" "instance" {
+  name                  = "terraform_test_rds_instance"
+  flavor                = "rds.pg.c2.medium"
+  availability_zone     = ["{{ availability_zone }}"]
+  security_group_id     = huaweicloud_networking_secgroup.secgroup.id
+  vpc_id                = "{{ vpc_id }}"
+  subnet_id             = "{{ subnet_id }}"
+  enterprise_project_id = "{{ enterprise_project_id }}"
+
+  db {
+    password    = "Huangwei!120521"
+    type        = "PostgreSQL"
+    version     = "10"
+    port        = "8635"
+  }
+  volume {
+    type = "ULTRAHIGH"
+    size = 50
+  }
+  backup_strategy {
+    start_time  = "08:00-09:00"
+    keep_days   = 1
+  }
+}
+
+resource "huaweicloud_rds_read_replica_instance" "replica_instance" {
+  name                  = "terraform_test_rds_read_replica_instance"
+  flavor                = "rds.pg.c2.medium.rr"
+  replica_of_id         = huaweicloud_rds_instance.instance.id
+  availability_zone     = "{{ enterprise_project_id }}"
```

Comment:
s/enterprise_project_id/availability_zone_id/

Reply:
Yes, I made a mistake, have fixed it.

Reply:
fixed